.changeset/cool-kiwis-shop.md

```diff
@@ -0,0 +1,12 @@
+---
+"@comet/admin-theme": major
+---
```

Comment:
One could also argue that this is not a major change, but since some defining properties (like the height) change, it felt more right to me to merge this into next